.github/workflows/check-lighthouse-schema.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2.4.0
+        with:
+          ref: ${{ github.head_ref }}
```

Comment:
I think we'd want to leave this as it was before? The checkout action is smart about this, and uses `GITHUB_REF` provided by GitHub, which already varies based on the trigger event: a merge commit for `pull_request` triggers, and HEAD ref on `push` triggers.

Did you mean to make this change, or were just copying from the previous actions README?

Reply:
Oh, I was just following the README and it worked so I didn't change anything.